app/services/v2/people/finder.rb

```diff
@@ -19,6 +19,14 @@ def split_params(name)
         filter_params[name]&.split(',')
       end
 
+      def split_and_upcase_params(name)
+        params = split_params(name)
+
+        return params&.map(&:upcase) if name == :prison_number
```

Comment:
It sounds strange to upcase the params only for `prison_number`,
also instead of upcase the filter, is it not better to perform a case insensitive search?

Reply:
Chatted with Alessandro and Dom about this. Prison numbers can never be downcased so this is fine. We're going to review the other identifiers separately to potentially simplify this method - stop special casing prison_number

Reply:
I suggest allowing a case insensitive search instead, by utilising the [CITEXT](https://www.postgresql.org/docs/9.1/citext.html) extension on Postgres instead?

Reply:
I don't really understand the advantage?

Reply:
Okay, I've been convinced. I'll make all the identifiers case insensitive. Thanks @alexdesi and @cwrw :)